.gitlab-ci/molecule.yml

```diff
@@ -29,10 +32,15 @@
     when: always
     paths:
     - molecule_logs/
+  cache:
+    key: $CI_JOB_NAME_SLUG
```

Comment:
Just to clarify : this is meant to cache between invocation of the same job on different pipelines runs, right, and not between jobs ?

Reply:
yes, 
Same job name = same cache
I want to avoid to have one massive cache blob for all jobs

Reply:
![image](https://github.com/user-attachments/assets/d7519b95-9d44-4c53-9950-b71dacda75ba)

keys looks like ^